torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6470,10 +6470,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            supports_out=False,
            supports_autograd=False,
-           skips=(
-               # JIT has issue when op is passed as lambda
-               SkipInfo('TestCommon', 'test_variant_consistency_jit'),
-           ),
+           has_fake_function=True,
```

Comment:
`resize_` is interesting. 

If `resize_` defined its inplace variant, would it be simpler to modify the jit test to simply acquire all the available variants, filter the function variant if it was "fake", and then have logic to account for whether the inplace variant was encountered?

The note in the test says it can't handle inplace inputs, but it looks like it is handling `resize_` in this PR?

Reply:
i think it can't handle inplace in scripting only..